### PR TITLE
Changes to support Android build errors

### PR DESCRIPTION
**Pull request by deerparker**
Few changes to allow Android build to complete.
- Bumped Android gradle versions to 5.4.1/3.5.2.
- reformatted build.gradle wrapper (as per https://stackoverflow.com/questions/53709282/cannot-add-task-wrapper-as-a-task-with-that-name-already-exists)
- added properties to support androidx in gradle.properties (which may be redundant with a later version which I will follow up on)
- added missing package dependency 'scripts' 

This might have just been my environment but I also had to run this to get past an error https://github.com/gatsbyjs/gatsby/issues/11406#issuecomment-458769756

Now builds the Android app (and a quick test that it didnt break web worked fine)

### CODE REVIEW

**Comment by deerparker** [comment]
Worth also saying, if it fails to run on the device first time, I had to also follow this manually afterwards - https://medium.com/@adityasingh_32512/solved-unable-to-load-script-from-assets-index-android-bundle-bdc5e3a3d5ff

**Comment by deerparker** [comment]
lock files in the pull request by mistake. you can ignore them!

File: yarn.lock [hunk 1]
```diff
@@ -1,13013 +0,0 @@
-# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (EraYaN) [inline yarn.lock]: Please keep yarn.lock in the history. It makes sure everyone is running the same versions of all the dependencies.

**Review by EraYaN — CHANGES_REQUESTED** [review]
Looks good except for the missing lock file.

File: yarn.lock (hunk 1 above)
Comment (deerparker) [reply yarn.lock]: yarn.lock added back in

**Comment by EraYaN** [comment]
One last thing, is that bundle required for Android? It seems like it’s generated code, if so please add it to the .gitignore file.

**Comment by deerparker** [comment]
> One last thing, is that bundle required for Android? It seems like it’s generated code, if so please add it to the .gitignore file.

That's the file removed and .gitignore updated.  The file is generated is linked to this comment I added above https://github.com/jellyfin/jellyfin-react-client/pull/37#issuecomment-559872237

**Review by EraYaN — APPROVED** [review]

**Comment by EraYaN** [comment]
Alright thanks for the PR.

**Pull request merged**